src/Microsoft.Tye.Core/ConfigModel/ConfigApplication.cs

```diff
@@ -21,10 +21,15 @@ public class ConfigApplication
 
         public string? Name { get; set; }
 
+        [YamlMember(Alias = "app")]
+        public string? Application { get; set; }
```

Comment:
Let's talk about this. I had intended `name` to be the application name. I'm not sure what this is for.